file.go

```diff
@@ -678,6 +678,9 @@ type ValidateOpts struct {
 
 	// UnequalAddendaCounts skips checking that Addenda Count fields match their expected and computed values.
 	UnequalAddendaCounts bool `json:"unequalAddendaCounts"`
+
+	// skipAllValidate can be set to disable all of validate actions that included im
+	SkipAllValidate bool `json:"skipAllValidate"`
```

Comment:
```suggestion
	SkipAllValidation bool `json:"skipAllValidation"`
```

Would that be a better name?